activiti-cloud-build/activiti-cloud-build-dependencies-parent/pom.xml

```diff
@@ -16,9 +16,21 @@
     <spring-cloud-function.version>3.2.3</spring-cloud-function.version>
```

Comment:
Could you upgrade `spring-cloud.version` to version `2021.0.2`, so we can remove the override of `spring-cloud-function`?

Reply:
Done